Spec/RealtimeClientPresence.swift

```diff
@@ -77,6 +77,36 @@ class RealtimeClientPresence: QuickSpec {
                 }
             }
 
+            // RTP6
+            context("subscribe") {
+
+                // RTP6b
+                it("with a single action argument") {
+                    let options = AblyTests.commonAppSetup()
+
+                    let client1 = ARTRealtime(options: options)
+                    defer { client1.close() }
+                    let channel1 = client1.channels.get("test")
+
+                    let client2 = ARTRealtime(options: options)
+                    defer { client2.close() }
+                    let channel2 = client2.channels.get("test")
+
+                    waitUntil(timeout: testTimeout) { done in
+                        channel1.presence.subscribe(.Update) { member in
+                            expect(member.action).to(equal(ARTPresenceAction.Update))
+                            expect(member.clientId).to(equal("john"))
+                            expect(member.data as? NSObject).to(equal("away"))
+                        }
+                        channel2.presence.enterClient("john", data: "online")
+                        channel2.presence.updateClient("john", data: "away")
+                        channel2.presence.leaveClient("john", data: nil)
+                        done()
```

Comment:
Nothing, actually, I somehow missed [this line](https://github.com/ably/ably-ios/pull/320/commits/dc09a31b1dbf0df065bbbf3e0100d9a4d6fdbb5f#diff-b7ac3256e97dbdaeec04a488430a2669R97)...